pubspec.yaml

```diff
@@ -1,5 +1,5 @@
 name: protobuf
-version: 0.10.2
+version: 0.10.3-dev
```

Comment:
Why not just 0.10.3?

Reply:
Are we ready to publish now? We could...

Reply:
Yes, I think we should.

Reply:
ack